src/components/editor/metaform-editor-right-drawer.tsx

```diff
@@ -47,24 +99,28 @@ const MetaformEditorRightDrawer: React.FC<Props> = ({
   };
 
   /**
-   * Updates metaform field VisiblityIf parameters
+   * Updates metaform field or section visibleIf parameters
    *
-   * @param newMetaformField new metaform field
-   * @param eventValue FieldCondition value
+   * @param newMetaformUpdateFieldOrSection metaform field or section
+   * @param value event condition equals value (kylla / ei)
```

Comment:
Why Finnish values?

Reply:
Because in metaform VisibleIf equals value is Kylla/ei. 
Maybe that should be changed?